packages/@o3r/schematics/src/utility/monorepo.ts

```diff
@@ -25,3 +29,49 @@ export function findConfigFileRelativePath(tree: Tree, files: string[], originPa
 export function isNxContext(tree: Tree) {
   return tree.exists('/nx.json');
 }
+
+/**
+ * Determine if we are in an monorepo context
+ *
+ * @param tree
+ */
+export function isMonorepoContext(tree: Tree) {
```

Comment:
This is a bit unclear.
Does "monorepo" means "workspace" in this context? Then you should check `nx.json` or `angular.json`.
Does "monorepo" means "multi packages"? then checking the `workspaces` field make sense but not checking the `nx.json`

Reply:
My intention was to detect multipackages. 
It's what I need if I want to apply changes from `ng add o3r core` at project root level. 
So the root rules will be applied when I run the `o3r create`, but also when I run the `ng add o3r core` in a project which I detect that it is not a multipackage.
So I'll stay with checking `workspaces` filed in package.json and rename the helper function.

Reply:
then you can remove the `isNxContext(tree) || `

Reply:
it's what I have done indeed :)